### PR TITLE
KREST-12278 Add service.yml to migrate to semaphore

### PR DESCRIPTION
**Pull request by trnguyencflt**

### CODE REVIEW

File: service.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  enable: true
+semaphore:
+  enable: true
+  pipeline_type: cp-dockerfile
+  docker_repos: ['confluentinc/cp-kafka-rest',]
+  maven_phase: 'package'
+  maven_skip_deploy: true
+  build_arm: true
+  cve_scan: true
+  sign_image: true
```
Comment (sriman-confluent) [inline service.yml:16]: I believe this needs `nano_version: true`. https://github.com/confluentinc/confluent-security-plugins/pull/666

**Review by msn-tldr — COMMENTED** [review]
lgtm, will wait for @sriman-confluent 's comment to be resolved.

**Review by sriman-confluent — APPROVED** [review]
LGTM, my comment is resolved

**Review by msn-tldr — APPROVED** [review]
Lgtm, Jenkins step is failing that should be fixed before merging.

Do you know why is it failing due missing base image?

**Comment by trnguyencflt** [comment]
Jenkins is having issue atm, the change isn't related to Jenkins so proceed to merge. If there is issue on master nightly, we can always revert this change.

**Pull request merged**